src/other/Table.js

```diff
@@ -1,12 +1,13 @@
 "use strict";
 (function (root, factory) {
     if (typeof define === "function" && define.amd) {
-        define(["d3", "../common/HTMLWidget", "./Paginator", "../common/Utility", "css!./Table"], factory);
+        define(["d3", "../common/HTMLWidget", "./Paginator", "../common/Utility", "../common/Widget", "css!./Table"], factory);
```

Comment:
@dtsnell4 unfortunately for instanceof to work it needs to include the Widget definition as you have done above.  So its technically correct (even if it does not feel like it).